modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/AbstractTypeScriptClientCodegen.java

```diff
@@ -386,6 +390,9 @@ public void processOpts() {
         if (additionalProperties.containsKey(CodegenConstants.ENUM_PROPERTY_NAMING)) {
             setEnumPropertyNaming((String) additionalProperties.get(CodegenConstants.ENUM_PROPERTY_NAMING));
         }
+        if (additionalProperties.containsKey(ENUM_PROPERTY_NAMING_REPLACE_SPECIAL_CHAR)) {
```

Comment:
please also add a new `cliOptions.add(new CliOption(` above to make sure this is properly registered and exposed in the docs

Reply:
and then please re-generate the docs and commit

Reply:
Hello @macjohnny, it's done 😃